.appveyor.yml

```diff
@@ -7,32 +7,11 @@ environment:
   PYTHONIOENCODING: "UTF-8"
 
   matrix:
-    # Note: Because we have to separate the py2 and py3 components due to compiler version, we have a race condition for non-python packages.
-    # Not sure how to resolve this, but maybe we should be tracking the VS version in the build string anyway?
     - TARGET_ARCH: "x86"
-      CONDA_PY: "27"
-      PY_CONDITION: "python >=2.7,<3"
-      CONDA_INSTALL_LOCN: "C:\\Miniconda"
-    - TARGET_ARCH: "x86"
-      CONDA_PY: "35"
-      PY_CONDITION: "python >=3.5,<3.6"
-      CONDA_INSTALL_LOCN: "C:\\Miniconda35"
-    - TARGET_ARCH: "x86"
-      CONDA_PY: "36"
-      PY_CONDITION: "python >=3.6"
       CONDA_INSTALL_LOCN: "C:\\Miniconda36"
+
     - TARGET_ARCH: "x64"
-      CONDA_PY: "27"
-      PY_CONDITION: "python >=2.7,<3"
       CONDA_INSTALL_LOCN: "C:\\Miniconda-x64"
```

Comment:
This should be Miniconda3-x64.

Reply:
Would you like to submit a follow-up PR to correct this?